pkg/operation/botanist/component/metricsserver/metrics_server.go

```diff
@@ -262,6 +268,9 @@ func (m *metricsServer) computeResourcesData() (map[string][]byte, error) {
 					managedresources.LabelKeyOrigin: managedresources.LabelValueGardener,
 					v1beta1constants.GardenRole:     v1beta1constants.GardenRoleSystemComponent,
 				}),
+				Annotations: map[string]string{
+					resourcesv1alpha1.PreserveResources: true,
```

Comment:
```
$ make install
> Install
# github.com/gardener/gardener/pkg/operation/botanist/component/metricsserver
pkg/operation/botanist/component/metricsserver/metrics_server.go:272:6: undefined: "github.com/gardener/gardener-resource-manager/pkg/apis/resources/v1alpha1".PreserveResources
pkg/operation/botanist/component/metricsserver/metrics_server.go:272:23: cannot use true (type untyped bool) as type string in map value
make: *** [install] Error 2
```

Reply:
I guess we need a new release (and revendoring afterwards) of `github.com/gardener/gardener-resource-manager`, right?